From 4a0017cfd6562e35f32e7a7839e1c3f9de6e955d Mon Sep 17 00:00:00 2001
From: Srujana Challa <schalla@marvell.com>
Date: Thu, 18 May 2023 16:13:24 +0530
Subject: [PATCH 462/955] ci: enable event mode inline IPsec autotest

This patch enables event mode inline IPsec autotest in CI.

Signed-off-by: Srujana Challa <schalla@marvell.com>
Change-Id: Idcd5d6be15ecb5d608d586372b1e8467e5b565e7
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/dataplane/dpdk/+/103681
Base-Builds: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Base-Tests: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Tested-by: sa_ip-toolkits-Jenkins <sa_ip-toolkits-jenkins@marvell.com>
Reviewed-by: Jerin Jacob Kollanukkaran <jerinj@marvell.com>
---
 .../cnxk-tests/inline_ipsec_autotest/inline_ipsec_autotest.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/marvell-ci/test/cnxk-tests/inline_ipsec_autotest/inline_ipsec_autotest.sh b/marvell-ci/test/cnxk-tests/inline_ipsec_autotest/inline_ipsec_autotest.sh
index 9c9fcfe079cee..c2462339886ef 100755
--- a/marvell-ci/test/cnxk-tests/inline_ipsec_autotest/inline_ipsec_autotest.sh
+++ b/marvell-ci/test/cnxk-tests/inline_ipsec_autotest/inline_ipsec_autotest.sh
@@ -15,6 +15,9 @@ ETHERNET_DEVICE="$ETH_DEV"
 CRYPTO_DEV=${CRYPTO_DEV:-$(lspci -d :a0f3 | head -1 | awk -e '{ print $1 }')}
 CRYPTO_DEVICE="$CRYPTO_DEV"
 
+SSO_DEV=${SSO_DEV:-$(lspci -d :a0f9 | tail -1 | awk -e '{ print $1 }')}
+EVENT_DEVICE="$SSO_DEV"
+
 TEST_TYPE=$1
 
 if [[ -f $SCRIPTPATH/../../../../app/test/dpdk-test ]]; then
@@ -56,6 +59,7 @@ run_inline_ipsec_tests() {
 
 #					DPDK TEST NAME		TEST ARGS
 register_cn10k_inline_ipsec_test	inline_ipsec_autotest	"-a $ETHERNET_DEVICE -a $NIX_INL_DEVICE -a $CRYPTO_DEVICE"
+register_cn10k_inline_ipsec_test	event_inline_ipsec_autotest	"-a $ETHERNET_DEVICE -a $NIX_INL_DEVICE -a $CRYPTO_DEVICE -a $EVENT_DEVICE"
 
 case $TEST_TYPE in
 	inline_ipsec_tests)
-- 
2.25.1

